CHANGELOG.md

```diff
@@ -2,15 +2,22 @@
 
 ## [Unreleased]
 
+## 0.1.0 (n/a)
+
 ### New Features
 
-- add --log-capture-to option to capture logs for log_proxy
+- Add `--log-capture-to` option to capture logs for `log_proxy`.
```

Comment:
the changelog is automatically generated from git commits (with conventional commit specification)

Reply:
Thanks for letting me know. So, I will revert all manual changes to the CHANGELOG.md file, right?